common/interface.h

```diff
@@ -0,0 +1,21 @@
+#ifndef __INTERFACE__
+#define __INTERFACE__
+
+#include <string>
+#include <net/if.h>
+
+namespace swss {
+
+inline bool isInterfaceNameLenOk(const std::string &ifaceName)
+{
+    if (ifaceName.length() >= IFNAMSIZ)
```

Comment:
@arfeigin why `>=` ?

`Constant: size_t IFNAMSIZ`
This constant defines the maximum buffer size needed to hold an interface name, including its terminating zero byte.

Reply:
Interface name length must be less than IFNAMSIZ as it takes into account also terminating zero, that's why I used >=